src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs

```diff
@@ -57,7 +57,7 @@ public WebApplicationTestFixture()
         /// <param name="solutionRelativePath">The path to the project folder relative to the solution file of your
         /// application. The folder of the first .sln file found traversing up the folder hierarchy from the test execution
         /// folder is considered as the base path.</param>
-        public WebApplicationTestFixture(string solutionRelativePath)
+        protected WebApplicationTestFixture(string solutionRelativePath)
```

Comment:
What's this about?

Reply:
xUnit doesn't know what constructor to pick

> and makes a couple of constructors protected so that WebApplicationTestFixture can be directly instantiated by xUnit